charts/incubator/hyperswitch-app/router.toml

```diff
@@ -0,0 +1,314 @@
+[bank_config]
+eps.adyen.banks = "bank_austria,bawag_psk_ag,dolomitenbank,easybank_ag,erste_bank_und_sparkassen,hypo_tirol_bank_ag,posojilnica_bank_e_gen,raiffeisen_bankengruppe_osterreich,schoellerbank_ag,sparda_bank_wien,volksbank_gruppe,volkskreditbank_ag"
```

Comment:
Let's add both sandbox and production toml file, based on user input we can choose one of it. By default sandbox will be chosen